src/pyxplor/plot_numeric.py

```diff
@@ -49,13 +52,68 @@ def plot_numeric(
 
     Returns
     -------
-    None
-
+    tuple
+        A tuple containing the `fig` and `ax` objects.
 
     Examples
     --------
     numeric = ["numeric_var1", "numeric_var2"]
-    plot_numeric(df, numeric, "hist+kde", 30, 10, (15, 20))
+    plot_numeric(df, numeric, "hist+kde", 30, 10, (10, 15))
     """
```

Comment:
i think we have to add input validation and raise error in case an invalid argument is provided